packages/cosmo/package.py

```diff
@@ -110,89 +111,89 @@ class Cosmo(MakefilePackage):
     conflicts('+cppdycore', when='%pgi cosmo_target=cpu')
 
     build_directory = 'cosmo/ACC'
+    
+    def setup_run_environment(self, env):
+        env.set('REAL_TYPE', self.spec.variants['real_type'].value.upper())
```

Comment:
I think it would be better to prefix this, maybe `COSMO_REAL_TYPE`?

Reply:
Yes but then I have to change the submit scripts as well and you know it is not that easy so if we could stay with that...